scripts/init_serverless_sdk.py

```diff
@@ -10,11 +10,11 @@
 import re
 
 import sentry_sdk
-from sentry_sdk._types import MYPY
+from sentry_sdk._types import TYPE_CHECKING
 from sentry_sdk.utils import Dsn
 from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
 
-if MYPY:
+if TYPE_CHECKING:
```

Comment:
should we still expose `MYPY` as an alias just in case someone's importing it currently?

Reply:
oh.... yeah. probably. they shouldn't do that, right?

Reply:
shouldn't.. but I would remove it in a major just to be safe